spec/models/recipe_spec.rb

```diff
@@ -12,4 +12,20 @@
     it {should have_many :recipe_ingredients}
     it {should have_many(:ingredients).through(:recipe_ingredients)}
   end
+
+  describe "instance methods" do
+    describe "#total_cost" do    
+      it "adds the total cost of all ingredients per recipe" do
+        enchiladas = Recipe.create!(name: "Enchiladas", complexity: 4, genre: "Tex-Mex")
+
+        ground_beef = Ingredient.create!(name: "Ground Beef", cost: 2)
+        cheese = Ingredient.create!(name: "Cheese", cost: 3)
+        tomatillo = Ingredient.create!(name: "Tomatillo", cost: 5)
+        tortilla = Ingredient.create!(name: "Tortilla", cost: 6)
+    
+        enchiladas.ingredients << [ground_beef, cheese, tomatillo, tortilla] 
+        expect(enchiladas.total_cost).to eq(16)
+      end
+    end
+  end
```

Comment:
Create another recipe and check that the total cost of that recipe is something other than 16. This ensures that your logic is dynamic and not just return 16 as a hard-coded value.